avahi-monitor.py

```diff
@@ -36,6 +38,16 @@
     # "_nfd._tcp",
     "_nfd._udp"]
 
+def face_uri_from_triplet(type, host, port):
+    if type == '_nfd._udp':
+        proto = 'udp4'
+    else:
+        proto = 'tcp4'
+    return "%s://%s:%s"%(proto, host, port)
```

Comment:
Nitpick: PEP 8 requires spaces around the `%`. Can you set up your editor to highlight PEP 8 violations?